package.json

```diff
@@ -83,6 +83,7 @@
     "react-ga": "^1.0.12",
     "react-router": "^1.0.3",
     "react-select": "^0.9.1",
+    "react-wp-content-loader": "git://github.com/mmmavis/react-wp-content-loader.git",
```

Comment:
let's not do this: even if it's alpha, just publish it to npm, and let's make package.json load it from there